android/src/main/java/com/oney/WebRTCModule/CameraCaptureController.java

```diff
@@ -184,7 +341,9 @@ private Pair<String, VideoCapturer> createVideoCapturer(String deviceId, String
 
         // Otherwise, use facingMode (defaulting to front/user facing).
         final boolean isFrontFacing = facingMode == null || !facingMode.equals("environment");
```

Comment:
Same as above, let's write this the straightforward way, not sure why I wrote it that way in the past 😅